ush/config_defaults.sh

```diff
@@ -352,12 +352,20 @@ WFLOW_LAUNCH_LOG_FN="log.launch_FV3LAM_wflow"
 # FCST_LEN_HRS:
 # The length of each forecast, in integer hours.
 #
+# DA_CYCLE_INTERV:
+# Data assimilation cycle interval
```

Comment:
This is mean to be an integer in hours? Maybe add that info?

Reply:
Added "in integer hours for now" because we only do hourly now. May be sub hourly will come soon.